percy/snapshot.py

```diff
@@ -81,3 +81,28 @@ def percy_snapshot(driver, name, **kwargs):
     except Exception as e:
         print(f'{LABEL} Could not take DOM snapshot "{name}"')
         print(f'{LABEL} {e}')
+
+# Take screenshot on driver
+def percy_automate_screenshot(driver, name, **kwargs):
+    if not is_percy_enabled(): return
+
+    try:
+        # Post to automateScreenshot endpoint with driver options and other info
+        response = requests.post(f'{PERCY_CLI_API}/percy/automateScreenshot', json={**kwargs, **{
+            'client_info': CLIENT_INFO,
+            'environment_info': ENV_INFO,
+            'sessionId': driver.session_id,
+            'commandExecutorUrl': driver.command_executor._url, # pylint: disable=W0212
+            'capabilities': dict(driver.capabilities),
+            'sessionCapabilites':dict(driver.desired_capabilities),
+            'snapshotName': name
```

Comment:
we need to pass options here as well which are given by the user.

Reply:
Already passing kwargs as we do in percy_snapshot https://github.com/percy/percy-selenium-python/blob/954c90a65352e2bce2bc5e452c73ace9e1252e63/percy/snapshot.py#L93